README.md

```diff
@@ -1,135 +1,131 @@
-# Firebase CLI & Web Frameworks
-
-## Frameworks
-
-| Framework  | Support | |
-| ---------- | ------- | - |
-| [Next.js](https://firebase.google.com/docs/hosting/frameworks/nextjs) | **Early preview** | |
-| [Angular](https://firebase.google.com/docs/hosting/frameworks/angular) | **Early preview** | |
-| [Express](https://firebase.google.com/docs/hosting/frameworks/express) | **Early preview** | |
-| Flask | **Early preview** | Coming soon... |
-| Django | Experimental | Coming soon... |
-| [Flutter](https://firebase.google.com/docs/hosting/frameworks/flutter) | Experimental | |
-| Nuxt | Experimental | |
-| Astro | Experimental | |
-| SvelteKit | Experimental | |
-| Preact<br>React<br>Lit<br>Svelte<br>and more... | Experimental | Static web apps, powered by *Vite* |
+# App Hosting adapters
 
 ## Overview
 
-Firebase Hosting integrates with popular modern web frameworks including Angular and Next.js. Using Firebase Hosting and
-Cloud Functions for Firebase with these frameworks, you can develop apps and microservices in your preferred framework
-environment, and then deploy them in a managed, secure server environment. Support during this early preview includes
-the following functionality:
+App Hosting provides no-config-needed build and deploy support for Web apps developed in these frameworks:
 
-* Deploy Web apps comprised of static web content
-* Deploy Web apps that use pre-rendering / Static Site Generation (SSG)
-* Deploy Web apps that use server-side Rendering (SSR)—full server rendering on demand
+* Next.js 13+
+* Angular 17.2+
 
-Firebase provides this functionality through the Firebase CLI. When initializing Hosting on the command line, you
-provide information about your new or existing Web project, and the CLI sets up the right resources for your chosen Web
-framework.
+This repo holds the code for the adapters that enable support for these frameworks. At a high level these adapters transform frameowkr specific configurations into an output bundle spec that App Hosting is able to process and configure frameworks support. For more information see our [documentation](https://firebase.google.com/docs/app-hosting/about-app-hosting#frameworks)
```

Comment:
Typo

Reply:
I think this may have been an old diff, not sure what happened but these were already addressed for me locally and in the pr I saw. I just pushed another change to address a couple other changes too so could you check again and see if these are fixed now?